src/components/seekbar-live-playback-container/seekbar-live-playback-container.js

```diff
@@ -71,20 +71,33 @@ class SeekBarLivePlaybackContainer extends Component {
       <SeekBar
         playerElement={this.props.playerContainer}
         showTimeBubble={this.props.showTimeBubble}
-        changeCurrentTime={time => (this.props.player.currentTime = time)}
+        changeCurrentTime={time => {
+          if (!this.props.player.isOnLiveEdge() || time < this.duration) {
```

Comment:
if !(this.props.player.isOnLiveEdge() && time === this.duration)

Reply:
fixed